osu.Framework/Graphics/Containers/BufferedContainer.cs

```diff
@@ -26,8 +26,8 @@ namespace osu.Framework.Graphics.Containers
     public class BufferedContainer : BufferedContainer<Drawable>
     {
         /// <inheritdoc />
-        public BufferedContainer(RenderbufferInternalFormat[] formats = null, bool pixelSnapping = false)
-            : base(formats, pixelSnapping)
+        public BufferedContainer(RenderbufferInternalFormat[] formats = null, bool pixelSnapping = false, bool clipToRootNode = false)
```

Comment:
```suggestion
        public BufferedContainer(RenderbufferInternalFormat[] formats = null, bool pixelSnapping = false, bool clipToRootNode = true)
```

I think we agreed that this is best left enabled by default.